Controller/Payment/FormDataAnalytics.php

```diff
@@ -78,6 +78,8 @@ public function execute()
 
             $trackResponse = $this->trackPluginInstrumentation->rzpTrackSegment($event, $properties);
 
+            $trackResponseDatalake = $this->trackPluginInstrumentation->rzpTrackDataLake($event, $properties);
```

Comment:
@abdulwahidsharief Please include tracking similar to Segment, check for function call `rzpTrackSegment` to know where events are tracked to Segment.